csharp/ql/test/library-tests/ir/ir/indexers.cs

```diff
@@ -0,0 +1,26 @@
+class Indexers
+{
+    public class Contact
+    {
+        private string[] address = new string[3];
+        public string this[int index]
+        {
+            get
+            {
+                return address[index];
+            }
+            set
+            {
+                address[index] = value;
+            }
+        }
+    }
+
+    public static void Main()
+    {
+        Contact contact = new Contact();
+        contact[0] = "Begumpet";
+        contact[1] = "Hyderabad";
+        contact[2] = "Telengana";
```

Comment:
add a test for `get_Item` as well? E.g. `contact[3] = contact[2]`.

Reply:
Added and modified test a bit. Also discovered a spurious `Load` when testing `get_Item`.